prog/vm/aws/nexus.rb

```diff
@@ -202,6 +202,16 @@ def before_run
         nap 0
       end
       raise
+    rescue Aws::EC2::Errors::Unsupported => e
```

Comment:
Should we also check if the error message matches somewhat to make sure its the instance type being unavailable in the AZ?

```
Your requested instance type (m8gd.large) is not supported in your requested Availability Zone (us-east-1e). Please retry your request by not specifying an Availability Zone or choosing us-east-1a, us-east-1b, us-east-1c, us-east-1d, us-east-1f.
```